src/views/Dashboard.vue

```diff
@@ -0,0 +1,29 @@
+<template lang="pug">
+  .dashboard-container
+    nav-bar
+</template>
+
+<script>
+import NavBar from './NavBar'
```

Comment:
Navbar should be in the `components` folder, not in views

Reply:
Yeah, was thinking the same but wasn't sure about the structure. Will create that folder and move this component to it.